docs/conf.py

```diff
@@ -25,6 +25,7 @@ def setup(sphinx):
     thisdir = os.path.dirname(os.path.realpath(__file__))
     sys.path.insert(0, thisdir + '/utils')
     from pygments_lexer_solidity import SolidityLexer
+    sphinx.add_stylesheet('css/custom.css')
```

Comment:
Can you move that to the bottom of the function? Everything else concerns setting up the lexer. Perhaps even add a blank line in between.